tools/faucet/webserver/web_server.go

```diff
@@ -170,3 +172,10 @@ func balanceReqHandler(faucetServer *faucet.Faucet) gin.HandlerFunc {
 		c.JSON(http.StatusOK, gin.H{"balance": balance.String()})
 	}
 }
+
+// returns the remaining native balance of the faucet
+func healthReqHandler() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		c.JSON(http.StatusOK, gin.H{"healthy": true})
+	}
+}
```

Comment:
Faucet didn't have a health handler - particularly useful to determine when the container is up and running.